microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,491 @@ ms.date: 05/04/2021
 
-  ![The Main section](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-main.msft.png)
+Scroll down to the **Diagnostics** section which says to **Minimize main-thread work** and **Reduce JavaScript execution time**. The main thread is where the browser does most of the work that's needed to display a page, such as:
 
-1. Scroll down to the bottom of the **Main** section.  When you use a framework, most of the upper activity is caused by the framework, which is usually out of your control.  The activity caused by your app is usually at the bottom.  In this app, it seems like a function named `App` is causing a lot of requests to a `mineBitcoin` function.  It sounds like Tony might be using the devices of his fans to mine cryptocurrency...
+* Parsing and running HTML.
+* Parsing and applying CSS to DOM elements. 
+* Running JavaScript.
 
-   ![Hover on the mineBitcoin activity](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-timings-minebitcoin.msft.png)
+In this case, it looks like the biggest bottleneck is that the page runs too much JavaScript code on page load.
 
-   > [!NOTE]
-   > Although the requests that your framework makes are usually out of your control, sometimes you might structure your app in a way that causes the framework to run inefficiently.  Restructuring your app to use the framework efficiently is a way to do less main thread work.  However, this requires a deep understanding of how your framework works, and what kind of changes you make in your own code in order to use the framework more efficiently.
+Use the **Performance** tool to analyze what work the main thread is doing while the page loads, and find ways to defer or remove unnecessary work:
 
-1. Expand the **Bottom-Up** section.  This tab breaks down what activities took up the most time.  If nothing is displayed in the **Bottom-Up** section, click the label for **Main** section.  The **Bottom-Up** section only shows information for whatever activity, or group of activity, you have currently selected.  For example, if you chose one of the `mineBitcoin` activities, the **Bottom-Up** section is only going to show information for that one activity.
+1. Open the **Performance** tool.
 
-   ![The Bottom-Up tab](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-timings-summary-minebitcoin.msft.png)
+1. In the **Performance** tool, click **Start profiling and reload the page** (![Refresh](../media/reload-icon.msft.png)), and then click **Stop** once the page has fully loaded.
 
-The **Self Time** column shows you how much time was spent directly in each activity.  For example, in the following figure, about 63% of main thread time was spent on the `mineBitcoin` function.
+    DevTools displays a visualization of all the work that the browser performed in order to load the page. This visualization is referred to as the **trace**:
```

Comment:
```suggestion
    DevTools displays a visualization of all the work that the browser performed in order to load the page. This visualization is referred to as the _trace_:
```
i need to double check style guides but i go w/ convention that terms being defined = italic.